src/common/Tile/GenreSection/styled.js

```diff
@@ -0,0 +1,17 @@
+import styled, { css } from "styled-components";
+
+export const StyledSection = styled.section`
```

Comment:
tu chyba styled.div powinno być bo nazywać się tak może ale semantycznie to zwykły div raczej, section to część całości strony a nie jednego elementu

Reply:
Ok, można zmienić na div ;)